src/components/ShowCard.js

```diff
@@ -0,0 +1,104 @@
+import { useState, useContext } from "react";
+import noImageFound from "../assets/noImageFound.jpg";
+import { black, black2, cyan1, cyan2, grey, white } from "../variables/colors";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faThumbsUp, faThumbsDown } from "@fortawesome/free-solid-svg-icons";
+import ShowsDataContext from "../contexts/ShowsDataContext";
+import styled from "styled-components";
+
+const ShowCard = (props) => {
+  const showObj = props.showObj;
+  const { id, image, language, name, summary } = showObj.show;
+
+  const [formattedSummary, setFormattedSummary] = useState(null);
+
+  // Triggers a modal to appear on the page with the movie details
+  const handleLoadMore = (summary) => {
+    const finalSummary = formatSummary(summary, summary.length);
+
+    setFormattedSummary(finalSummary);
+  };
+
+  const formatSummary = (summary, maxCharLength = 50) => {
+    let finalSummary = null;
+
+    // Check if summary is null or not
+    if (summary === null || summary === undefined) {
+      finalSummary = "N/A";
+    } else {
+      // Remove HTML tags
+      const regex = /(<([^>]+)>)/gi;
+      const cleanSummary = summary.replace(regex, "");
+
+      // Trim the summary
+      if (cleanSummary.length > maxCharLength) {
+        finalSummary = cleanSummary.substring(0, maxCharLength) + "...";
+      } else {
+        finalSummary = cleanSummary;
+      }
+    }
+
+    return finalSummary;
+  };
+
+  const score = Math.floor(showObj.score * 100);
+  const finalSummary = formatSummary(summary);
+
+  return (
+    <Card key={id}>
+      {image ? (
+        <img src={image.medium} alt={`Poster of ${name}`} />
+      ) : (
+        <img src={noImageFound} alt="No image found" />
+      )}
+      <Button>
```

Comment:
May want to use the IconBtn from the ./Buttons component file and then add custom styles if you like